### ISSUE TITLE
Bad pattern in oval:ssg-obj_password_pam_pwquality_ucredit:obj:1

### ISSUE THREAD

**Issue by solind** [OpenSCAP/scap-security-guide#3125]
#### Description of problem:

I believe there's an issue with a regular expression pattern in oval:ssg-obj_password_pam_pwquality_ucredit:obj:1

```
^ucredit[s\]*=[\s]*(-?\d+)(?:[\s]|$)
          ^
```

Notice the placement of the escape character above (which is how the pattern is currently written). This makes the first character class unclosed, causing the pattern to fail to parse in Java's regular expression parser.

I believe the intention behind the content was to permit white-space between the ucredit key and the equals sign.  That would mean re-writing the pattern as follows:

```
^ucredit[\s]*=[\s]*(-?\d+)(?:[\s]|$)
         ^
```

#### SCAP Security Guide Version:

0.1.39

#### Operating System Version:

CentOS, RHEL

**Comment by mpreisler** [comment]
looks like a typo. are you interested in creating a pull request so that you can get credit for this fix?

**Comment by solind** [comment]
@mpreisler Unfortunately I haven't been able to determine where this pattern is coming from, despite the using find, xargs and grep!

**Comment by mpreisler** [comment]
It's coming from https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/templates/template_OVAL_accounts_password#L51

but it looks like this has already been fixed

**Comment by solind** [comment]
LOL, I did find that, but concluded it couldn't be the source of the error because it's correctly escaped!

The release is 937 commits behind the master branch and I've been using the release ZIP... I guess I should be building the content.

**Comment by yuumasato** [comment]
Yes, the typo was fixed as side effect of templating done in #3045.
